docs/templates/function.mustache

```diff
@@ -9,7 +9,7 @@
 ```
 
 {{# has_description }}
-{{ get_description }}
+{{{ get_description }}}
```

Comment:
I was wondering if there was a way to exclude whatever was within the code tags to not be HTML-escaped, but it seems that's not possible, so I decided to go with unescaping the whole description instead.

Reply:
The only concern there would be if there is markup inside the description, which is not inside of code blocks. For example, I can see that in `amp_is_dev_mode()` this is now causing a problem since there is a `<html>`:

https://github.com/ampproject/amp-wp/blob/4d27de946d5c4aaee0a1c246dca67a89469cf968/includes/amp-helper-functions.php#L1404-L1408

This is getting rendered as:

![image](https://user-images.githubusercontent.com/134745/103305305-a4f47f00-49bf-11eb-8500-b62054cd8d5e.png)

This appears to be the only instance of that problem, so the fix may be to just wrap it in backticks:

```diff
--- a/includes/amp-helper-functions.php
+++ b/includes/amp-helper-functions.php
@@ -1414,7 +1414,7 @@ function amp_get_content_embed_handlers( $post = null ) {
 /**
  * Determine whether AMP dev mode is enabled.
  *
- * When enabled, the <html> element will get the data-ampdevmode attribute and the plugin will add the same attribute
+ * When enabled, the `<html>` element will get the data-ampdevmode attribute and the plugin will add the same attribute
  * to elements associated with the admin bar and other elements that are provided by the `amp_dev_mode_element_xpaths`
  * filter.
  *
```

That then produces the desired result:

![image](https://user-images.githubusercontent.com/134745/103305403-e553fd00-49bf-11eb-9408-d620859d9f2c.png)

The other impact of this change to discontinue escaping is double quotes and single quotes are no longer getting escaped:

![image](https://user-images.githubusercontent.com/134745/103305449-074d7f80-49c0-11eb-8e14-e2228cddc507.png)

![image](https://user-images.githubusercontent.com/134745/103305460-0f0d2400-49c0-11eb-8748-08a00eef22ad.png)

And this is a good thing!

Reply:
Applied this patch in 324ef5e6dfe2e7ac6baa7d498cee2fb77fb3b8a3.